packages/labs/react/src/test/create-component_test.tsx

```diff
@@ -250,6 +255,26 @@ suite('createComponent', () => {
     const onBar = (e: Event) => {
       barEvent = e;
     };
+
+    const onCustomBar = (e: CustomEvent<string>) => {
```

Comment:
I don't think we need a separate check for `CustomElement`, just narrow the type of event used in `onFoo` and check that a property in the event type but not in `Event` is accessible without a type-error.

Reply:
sounds good!

Reply:
I can't get the visual behavior to happen
I think this is more about what types the labs package narrows, not what events can be added.

https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts#L5113
https://github.com/microsoft/TypeScript/blob/main/lib/lib.dom.d.ts#L3693-L3700

Reply:
However, I _can_ get the visual behavior when I remove this file's CustomEventListener

Reply:
removed new tests.